blocks/multicolor-accordion/multicolor-accordion.js

```diff
@@ -0,0 +1,53 @@
+export default async function decorate(block) {
+  const accordionItems = block.querySelectorAll(':scope > div > div');
+  accordionItems.forEach((accordionItem) => {
+    const nodes = accordionItem.children;
+
+    const titleText = nodes[0];
```

Comment:
taking the first line feels a bit random, shouldn't we use a headline for marking the title?

Reply:
did you just close this without change?

Reply:
now its following column wise structure so node[0] is the first column its a title.